wordpress-mobile/WordPress-iOS
# Replacing the BlogDetails icons with tinted gridicons.

Small step towards reusing gridicons throughout the app, and in this round within `BlogDetailsViewController`.

To test:

1. Build and run the branch `feature/blog-details-gridicons` on either the device or simulator.
2. Ensure each icon is the correct icon for the corresponding row.

Needs review: @frosty would you take a peak at the code/file changes?
Needs review: @mattmiklic would you do a quick design review?

![simulator screen shot mar 31 2016 3 23 36 pm](https://cloud.githubusercontent.com/assets/1873422/14189847/22ee669a-f755-11e5-81a0-c9e94d54b051.png)
